a11y/Dockerfile

```
@@ -0,0 +1,39 @@
+FROM node:15.0.1
+
+LABEL maintainer="daniel.walford@ons.gov.uk"
+
+# Explicitly declare npm user is root (docker runs as root anyway) allows global package installation for npm, needed for @axe-core/cli
+RUN npm --global config set user root
+
+# Update apt then install wget to set up the Personal Package Archive (PPA) and unzip to install the Chromedriver
+RUN apt update \
+    && apt install -y wget unzip
+
+# Set up the Chrome PPA and update the package list and install chrome
+RUN wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add - \
+    && echo 'deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main' | tee /etc/apt/sources.list.d/google-chrome.list \
+    && apt update \
+    && apt install -y google-chrome-stable
```

See comment above about keeping image size down when using apt.

```suggestion
    && apt update \
    && apt install -y --no-install-recommends google-chrome-stable \
    && rm -rf /var/lib/apt/lists/*
```